lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -422,7 +437,7 @@ import Test.QuickCheck
 import Test.QuickCheck.Arbitrary.Generic
     ( genericArbitrary, genericShrink )
 import Test.QuickCheck.Extra
-    ( chooseNatural, report )
+    ( chooseNatural, report, shrinkBoundedEnum )
```

Comment:
This reminds me, I have a PR on `quickcheck` to add this function: https://github.com/nick8325/quickcheck/pull/350